.github/workflows/commitlint.yaml

```
@@ -0,0 +1,25 @@
+name: Lint Commit Messages
+
+on:
+  pull_request:
+    types: [opened, synchronize, edited]
+
+jobs:
+  lint-commits:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)